_posts/2001-11-01-drupal-plugin.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: help
+title:  "The Drupal Plugin"
+categories:
+- help
+- drupal-plugin
+---
+
+If you wish to translate the content of a Drupal website, you can install the Drupal module TMGMT Zanata to help send text and translations between Drupal and Zanata.
+
+For full installation, setup and usage instructions, visit [TMGMT Zanata | Drupal.org](https://www.drupal.org/project/tmgmt_zanata)
```

Comment:
I think the features are pretty much covered by "help send text and translations between Drupal and Zanata" - that is pretty much all it does right now. I guess maybe segmentation would be worth mentioning.